fipy/meshes/gmshMesh.py

```diff
@@ -835,7 +835,7 @@ def read(self):
 
             self.physicalFaceMap = nx.zeros(facesToV.shape[-1:], 'l')
             self.geometricalFaceMap = nx.zeros(facesToV.shape[-1:], 'l')
-            for face in facesDict.keys():
+            for face in list(facesDict.keys()):
```

Comment:
These changes to `for foo in list(bar.keys()):` seem strange and non-idiomatic. The [cheat-sheet for futurize](http://python-future.org/compatible_idioms.html#iterating-through-dict-keys-values-items) says that `for foo in bar:` is the proper way to write this in Python 2 and 3.